ssm-client.spec

```diff
@@ -8,8 +8,8 @@ Version:        %{_version}
 Release:        1%{?dist}
 Group:          Applications/Databases
 License:        AGPLv3
-Vendor:         Percona LLC
-URL:            https://percona.com
+Vendor:         Shattered Silicon
+URL:            https://shatteredsilicon.net
```

Comment:
@gordan-bobic 
I'm not very sure about this, is it legal to change it to ours info? And similar one in deb control file -> https://github.com/shatteredsilicon/ssm-client/pull/14/files#diff-2a13f1344504379e877324d3a0375adcbcb4cb27d7e575ad8f4618a52d6a00e0R4

If it's not legal, I'll change it back

Reply:
Other packages (ssm-manage, ssm-server etc) don't have **Vendor** and use github repo url as the **URL**, not sure why they made a difference in this one.

Reply:
Sorry, forgot to quote the removed lines, full diff is here:

```diff
- Vendor:         Percona LLC
- URL:            https://percona.com
+ Vendor:         Shattered Silicon
+ URL:            https://shatteredsilicon.net
```